src/views/preview/comment/compose-comment.jsx

```diff
@@ -147,6 +147,13 @@ class ComposeComment extends React.Component {
         this.setState({
             muteOpen: false
         });
+
+        // Cancel (i.e. complete) the reply action if the user clicked on the reply button while
+        // alreay muted. This "closes" the reply.  If they just got muted, we want to leave it open
+        // so the blue CommentingStatus box shows.
+        if (this.props.isReply && this.state.status !== ComposeStatus.REJECTED_MUTE) {
```

Comment:
One really subtle thing that is NOT worth fixing. When we show the modal, we still add some padding that would have surrounded the input, and then remove it when we close the modal. Unless you're looking at what's behind the modal, you'd never notice.